custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -143,6 +143,7 @@ def get_person_case_structure(self):
                 'update': {
                     'age': self.patient_detail.page,
                     'age_entered': self.patient_detail.page,
+                    'contact_phone_number': '91' + validate_phone_number(self.patient_detail.pmob),
```

Comment:
I'd see a message in the migration logs that migrating the particular user failed due to a phone number issue.  fwiw that hasn't happened yet in the 6 districts we've migrated.